Microsoft.Research/Contracts/MsCorlib/System.Threading.Tasks.Task.cs

```diff
@@ -288,14 +294,26 @@ public static TaskFactory Factory
     //
     // Returns:
     //     true if the task has completed due to being canceled; otherwise false.
-    //public bool IsCanceled { get; }
+    public bool IsCanceled
+    {
+      get
+      {
+        return default(bool);
+      }
+    }
     //
     // Summary:
     //     Gets whether this System.Threading.Tasks.Task has completed.
     //
     // Returns:
     //     true if the task has completed; otherwise false.
-    //public bool IsCompleted { get; }
+    public bool IsCompleted
```

Comment:
Potentially we can add a postcondition that the result is True when Status is in specified range of values.